File: app/src/main/java/com/neome/api/core/user/msg/MsgDeviceGet.java
// neome.ai API - do not change
//

package com.neome.api.core.user.msg;

import com.neome.api.core.base.msg.MsgVersion;
import com.neome.api.meta.base.Types.DeviceId;

@SuppressWarnings("unused")
public class MsgDeviceGet extends MsgVersion
{
  public DeviceId deviceId;
}

File: app/src/main/java/com/neome/api/core/user/msg/MsgDeviceGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.user.msg

import com.neome.api.meta.base.Types.DeviceId
import com.neome.api.core.base.msg.MsgVersion

interface MsgDeviceGet : MsgVersion
{
  val deviceId: DeviceId
}